packages/matchbox/src/components/Slider/Slider.js

```diff
@@ -12,7 +12,7 @@ function Slider(props) {
   const { defaultValue, disabled, id, max, min, onBlur, onFocus, onChange, precision, ticks, value } = props;
 
   const windowSize = useWindowSize(50);
-  const [sliderValue, setSliderValue] = React.useState(value || defaultValue);
+  const [sliderValue, setSliderValue] = React.useState(value || defaultValue != null ? defaultValue : min);
```

Comment:
Would this not work? (instead of using `!=null`). I might be missing something.
```
const [sliderValue, setSliderValue] = React.useState(value || defaultValue ? defaultValue : min);
```

Reply:
So i think I was trying to capture: If defaultValue is set, use it, otherwise use min. This allows `0` to still be valid. The bug was essentially that on first mount, `defaultValue` would sometimes be out of range.